commandapi-annotations-reloaded/src/main/java/dev/jorel/commandapi/annotations/reloaded/modules/base/CommandsClassJavadocGenerator.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright 2018, 2020 Jorel Ali (Skepter) - MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *******************************************************************************/
+package dev.jorel.commandapi.annotations.reloaded.modules.base;
+
+import dev.jorel.commandapi.annotations.reloaded.generators.CodeGenerator;
+import dev.jorel.commandapi.annotations.reloaded.generators.IndentedWriter;
+import dev.jorel.commandapi.annotations.reloaded.modules.commands.CommandRegisterMethodGeneratorContext;
+
+/**
+ * Generates the Javadoc for the Commands class
+ */
+public class CommandsClassJavadocGenerator implements CodeGenerator<CommandsClassGeneratorContext> {
+	@Override
+	public void generate(IndentedWriter out, CommandsClassGeneratorContext context) {
+		out.printOnNewLine("/**");
+		out.printOnNewLine(" * This class was automatically generated by the CommandAPI");
+		out.printOnNewLine(" * Generation time: %tc".formatted(context.generatorStarted()));
+		out.printOnNewLine(" * @Command classes used:");
+		for (CommandRegisterMethodGeneratorContext methodContext : context.registerMethods()) {
+			out.printOnNewLine(" * - {@link %s}".formatted(methodContext.qualifiedCommandClassName()));
+		}
```

Comment:
I personally think that generation time and classes used are irrelevant. People are probably not gonna look inside the generated classes and if they do, this information is not important to them.
The only relevant part would be that the class was generated.

Reply:
Unless a class was missing, I guess? I don't know, I just refactored what was already there. @JorelAli do you recall why it's like this?